recipes/wasmtime/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration, ConanException
+import os
+
+
+class WasmtimeConan(ConanFile):
+    name = 'wasmtime'
+    homepage = 'https://github.com/bytecodealliance/wasmtime'
```

Comment:
```suggestion
    homepage = 'https://wasmtime.dev/'
```

Reply:
Disagree, because later I will add package for `wasmtime-cpp` and it would be weird to have two projects with the same home directory

Reply:
Those should be the same recipe.

Reply:
It would be separate receipts `wasmtime` (only C API) and `wasmtime-cpp` (C++ API)

Reply:
Also `https://wasmtime.dev/` is not descriptive enough for understanding how to use this package

Reply:
You need to delete the C++ setting then https://github.com/conan-io/conan-center-index/blob/3d585c9d7bd6171fc61e277911926888f6c377a4/recipes/unqlite/all/conanfile.py#L30-L32

Reply:
In this recipt no compiler option ... You asked me to delete it ;)
```console
ERROR: wasmtime/0.28.0: Error in configure() method, line 43
        del self.settings.compiler.libcxx
        ConanException: 'settings.compiler' doesn't exist
'settings' possible configurations are ['arch', 'os']
```

Reply:
C++ uses a different repository: https://github.com/bytecodealliance/wasmtime-cpp

Reply:
No, you will still need this package
As I mentioned previously I will create support with dependency on this package when I will create `wasmtime-cpp`
See the issue:
https://github.com/bytecodealliance/wasmtime-cpp/issues/1

Reply:
It's okay to have two recipes share the same homepage.

Reply:
Home page should be meaningful
https://wasmtime.dev/ does not

Reply:
Perhaps you're right.
The c api lives in the `main/crates/c-api` subfolder.

How does this project relate to `wasm-c-api` at https://github.com/WebAssembly/wasm-c-api/?

Reply:
@maddanio It is unrelated project